### ISSUE TITLE
AddOn 'KkthnxUI' tried to call the protected function oUF_KkthnxPlayer:SetSize()

### ISSUE THREAD

**Issue by Kkthnx** [Kkthnx/KkthnxUI_Legion#103]
```
1x [ADDON_ACTION_BLOCKED] AddOn 'KkthnxUI' tried to call the protected function 'oUF_KkthnxPlayer:SetSize()'.
!BugGrabber\BugGrabber.lua:573: in function <!BugGrabber\BugGrabber.lua:573>
[C]: in function `SetSize'
...ddOns\KkthnxUI\Modules\Unitframes\Layouts\Layout.lua:236: in function <...ddOns\KkthnxUI\Modules\Unitframes\Layouts\Layout.lua:232>
(tail call): 
```

**Comment by Kkthnx** [comment]
If anyone has feedback about this error or can give a helping hand please do.

**Comment by Kkthnx** [comment]
Right now all we know about this, is this randomly can happen when coming off a vehicle.

**Comment by Kkthnx** [comment]
We might have fixed this but it needs testing. The fix is live on our 6.10 branch.

https://github.com/Kkthnx/KkthnxUI_Legion/tree/6.10

**Comment by Kkthnx** [comment]
Fixed
